### PR TITLE
Multi-GPU in-RAM

### PR DESCRIPTION
**Pull request by ilkarman**
Reran notebooks to consider also in-ram data to assess (and report)
performance of pure model and identify bottleneck from IO

### CODE REVIEW

**Comment by ilkarman** [comment]
Also TF model now supports NCHW and test with `fused=True` for slim.batchnorm

**Comment by ilkarman** [comment]
Tensorflow is faster than Keras(TF) - as expected. But something is up with the data-loader -> huge bottlenecking for all but Keras.
